streams/functions/src/main/java/com/hortonworks/streamline/streams/udf/Ltrim.java

```diff
@@ -0,0 +1,19 @@
+package com.hortonworks.streamline.streams.udf;
+
+import com.hortonworks.streamline.streams.rule.UDF;
+
+/**
+ * Trims leading white spaces from the input.
+ * See {@link String#trim()}
+ */
+public class Ltrim implements UDF<String, String> {
+    @Override
+    public String evaluate(String input) {
+        int len = input.length();
+        int st = 0;
+        while ((st < len) && (input.charAt(st) <= ' ')) {
```

Comment:
OK. I just found how String.trim is implemented.  http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/lang/String.java#l2852
Thanks for the pointer.